jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/enums/DimensionUnit.java

```diff
@@ -0,0 +1,13 @@
+package org.dcsa.jit.persistence.entity.enums;
```

Comment:
Since this is in OVS as well, perhaps we should think about moving it to shared-kernel at some point